src/sirhelpers.c

```diff
@@ -314,25 +314,24 @@ int _sir_fopen(FILE* restrict* restrict streamptr, const char* restrict filename
 
 struct tm* _sir_localtime(const time_t* restrict timer, struct tm* restrict buf) {
     if (_sir_validptr(timer) && _sir_validptr(buf)) {
-#if defined(__HAVE_STDC_SECURE_OR_EXT1__)
+#if defined(__HAVE_STDC_SECURE_OR_EXT1___)
 # if defined(__WIN__)
         errno_t ret = (errno_t)localtime_s(buf, timer);
```

Comment:
This shouldn't need a cast to `errno_t` on windows; that is the type it returns... and according to [cppreferfence](https://en.cppreference.com/w/c/chrono/localtime), on other platforms it should return `struct tm*` and the return value is reverse in meaning. NULL (0) means it failed, whereas [on windows](https://learn.microsoft.com/en-us/cpp/c-runtime-library/reference/localtime-s-localtime32-s-localtime64-s?view=msvc-170), 0 means success. Thanks MSFT.

Reply:
Fixed